api/src/certification/shared/domain/constants.js

```diff
@@ -0,0 +1,7 @@
+export const CERTIFICATION_FEATURES = {
+  CAN_REGISTER_TO_COMPLEMENTARY_CERTIFICATION_ALONE: {
+    key: 'CAN_REGISTER_TO_COMPLEMENTARY_CERTIFICATION_ALONE',
+    description:
+      "Permet l'accès pour un centre de certification à la nouvelle modale qui permet d'inscrire un candidat à une complementaire seule.",
```

Comment:
Je trouve ça trop orienté sur la modale :o 

Est-ce qu'on ne devrait pas partir sur du plus global, du genre "Permet le passage d'une certification Pix coeur ou Pix+ seul"

Reply:
Ca irait cette reformulation ?

```suggestion
      "Permet l'accès pour un centre de certification à l'inscription d'un candidat passant une complementaire seule.",
```

On voulait bien garder la notion de déblocage de l'inscription dans ce toggle. Car les autres features de cette Epix en principe devront découler de règles métiers et non pas du toggle.

Un exemple pour illustrer cela : il faudrait par exemple que "Ajouter la mention “(seule)” après le nom de la certification complémentaire" s'affiche pas parce qu'on a le flag, mais parce que le système a détectté que on était sur une inscription de type "seul" (je ne sais pas si c'est super clair 😅 )